### ISSUE TITLE
Exclude Non-English Content from Raw Dataset

### ISSUE THREAD

**Issue by dominic0df** [amosproj/amos2024ss08-cloud-native-llm#38]
## User story
1. As Machine Learning Engineer
2. I want to exclude non-english content from the raw dataset
3. So that my dataset becomes easier to handle and Q&A generation will become easier

## Acceptance criteria
* Non-english content is filtered out after the extraction of the data before it is loaded to the raw dataset in Hugging Face

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
